views/js/component/timeHandling/timeHandling.tpl

```diff
@@ -56,6 +56,11 @@
                     <input type="text" id="input-extra-time" data-control="time" value="{{time}}" maxlength="4" size="4" />
                     <label for="input-extra-time">{{__ "minutes"}}</label>
                 {{/if}}
+                <div class="errors">
+                    <ul>
+                        <div class="feedback-error small hidden">{{errorMessage}}</div>
+                    </ul>
```

Comment:
**question:** is this mandatory? The error messages look to be inserted on the fly. This is rendered hidden, and I don't see code to show it...

Moreover, the semantic is wrong: `div` as a child of `ul`... Without mentioning the `ul` is not targeted by the JavaScript...